src/utils/database-common/tests/tests/test_entries_streamer.rs

```diff
@@ -1,414 +0,0 @@
-// Copyright Kamu Data, Inc. and contributors. All rights reserved.
```

Comment:
Note for readers: duplicate file -- looks like a sad artifact of the previous my big merge.